### PR TITLE
Use ensure_packages

### PR DESCRIPTION
**Pull request by alexjfisher**
Trivial (famous last words) refactor.

### CODE REVIEW

File: manifests/packages.pp [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  if $minecraft::manage_java {
+  if $::minecraft::manage_java {
     class { '::java':
       distribution => 'jre',
       version      => 'latest',
     }
   }
 
-  ensure_resource('package', 'screen', {'ensure' => 'present' })
+  ensure_packages('screen')
```
Comment (alexjfisher) [inline manifests/packages.pp]: I'd be happy to wrap this in an `if $::minecraft::manage_screen` as well if required.

**Comment by bbriggs** [comment]
So this would resolve #12?

**Comment by alexjfisher** [comment]
@bbriggs It's probably easy enough for you to fix your base profile to also use `ensure_packages`.  Users of https://github.com/dhoppe/puppet-screen would probably still benefit from a `manage_screen` boolean parameter.

**Comment by juniorsysadmin** [comment]
I don't like the idea of managing Java within a module at all.

**Pull request merged**